src/app/tabs/run-tab.js

```diff
@@ -243,6 +244,59 @@ function updateAccountBalances (container, appAPI) {
 }
 
 /* ------------------------------------------------
+    RECORDER
```

Comment:
Is that possible to put this in a separate file?

Reply:
It's ...probably possible.
The `recorder.js` is already seperate.
The `run-tab.js` has buttons on it that use the recorder.

This logic is only about the buttons and where they are positioned and what they do.

I initially started to give the `recorder.js` a `.render()` method, but then felt, the design might very much depend on the panel and how to position the buttons inside the panel.

If you really think it should be put into a seperate file? should i reconsider the `.render()` or something else?

**implication:**
It would mean, i also need to pass or require `modelDialog` and `copy2clipboard` to recorder.

Reply:
so to answer your question: **no** (i don't have a good idea how to do that)

*(but yes, you can always make a separate file and put stuff in - if you have a good idea then just let me know)*